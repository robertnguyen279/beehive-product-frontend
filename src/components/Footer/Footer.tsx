import React from 'react';

export default function Footer(): React.ReactElement {
  return <div>Footer</div>;
}
