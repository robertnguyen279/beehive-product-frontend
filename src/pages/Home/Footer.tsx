import React from 'react';
import { useTranslation } from 'react-i18next';

const Footer = () => {
  const { t } = useTranslation('HomePage');
  return (
    <div className="home__footer relative z-10 py-5 text-white text-sm flex justify-center">
      <div className="px-3 cursor-pointer md:pt-16">{t('footer-link-home')}</div>
      <div className="px-3 cursor-pointer md:pt-16">{t('footer-link-aboutus')}</div>
      <div className="px-3 cursor-pointer md:pt-16">{t('footer-link-faqs')}</div>
      <div className="px-3 cursor-pointer md:pt-16">{t('footer-link-blog')}</div>
      <div className="px-3 cursor-pointer md:pt-16">{t('footer-link-contact')}</div>
    </div>
  );
};

export default Footer;
