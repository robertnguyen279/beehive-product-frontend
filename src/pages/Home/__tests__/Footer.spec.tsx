import React from 'react';
import { shallow } from 'enzyme';

import Footer from '../Footer';
// const clickFn = jest.fn();

describe('Footer', () => {
  const component = shallow(<Footer />);

  it('should render correctly', () => {
    expect(component).toMatchSnapshot();
  });
});
