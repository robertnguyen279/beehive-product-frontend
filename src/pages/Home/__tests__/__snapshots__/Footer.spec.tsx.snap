// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Footer should render correctly 1`] = `
<div
  className="home__footer relative z-10 py-5 text-white text-sm flex justify-center"
>
  <div
    className="px-3 cursor-pointer md:pt-16"
  >
    footer-link-home
  </div>
  <div
    className="px-3 cursor-pointer md:pt-16"
  >
    footer-link-aboutus
  </div>
  <div
    className="px-3 cursor-pointer md:pt-16"
  >
    footer-link-faqs
  </div>
  <div
    className="px-3 cursor-pointer md:pt-16"
  >
    footer-link-blog
  </div>
  <div
    className="px-3 cursor-pointer md:pt-16"
  >
    footer-link-contact
  </div>
</div>
`;
